front/app/containers/LandingPage/SignedInHeader.tsx

```diff
@@ -85,6 +85,10 @@ const HeaderImage = styled(Image)`
   width: 100%;
   height: auto;
 
+  ${media.biggerThanMaxTablet`
```

Comment:
Why would you only apply the filter on a screen bigger than a big tablet (>1023px)? Wouldn't it be better to always apply it? Or definitely from biggerThanSmallTablet already.

Also, could you add a comment to explain why this CSS is here?

Reply:
From speaking to @benfraserdesign the idea was that images used in the 2-column layout would look fine on smaller screens/devices, but on a widescreen logged-in banner a smaller image that looked fine in the 2-column logged out banner might look stretched. So we decided on blurring it at wider sizes but leaving it as-is on smaller screens, where it should look okay. I can see how that might be kind of funky though, it might be better to just do a slight blur constantly. Maybe to be fancy it could be 1px on small devices and 2px on tablets and larger.

For the breakpoint I matched it against the one below for the same selector, which works on `smallerThanMaxTablet`, but if `biggerThanSmallTablet` makes more sense I can change it to that. 

I'll add a comment too 👍